### ISSUE TITLE
PID variables in EEPROM

### ISSUE THREAD

**Issue by rwilbur** [inceptionev/FixMoreLungs#7]
We will need the variables to be stored in EEPROM so that in case of a reboot we still have the correct values for the variables

**Comment by lee-matthews** [comment]
Indeed, this functionality shall be provided by the eeprom module in the software.
